iib/web/api_v1.py

```diff
@@ -433,6 +433,9 @@ def patch_request(request_id):
             bundle_img = Image.get_or_create(bundle)
             bundle_img.operator = operator_img
 
+    if 'distribution_scope' in payload:
```

Comment:
Don't you need to validate the value?

Reply:
We already validate it in build.py. So I don't think we need to do it here as well.

Reply:
Sure, there is a precedent to validate all values on this API endpoint though. The main reason is that the API is separate from the workers, and trust should be limited.

Reply:
So L377 should throw an error if it's a non-empty string. Are you suggesting to validate something else too?